pylintrc

```diff
@@ -1,5 +1,5 @@
 [MASTER]
-ignore=tests,generated,vendored_sdks,privates
+ignore=tests,generated,vendored_sdks,privates,aaz
```

Comment:
Will this also make `src/azure-cli-core/azure/cli/core/aaz` unchecked?

Reply:
I'm just testing first to see if I can improve the test speed.
These parameters are more complicated than we think.
The ignore parameter represents `Files or directories to be skipped. They should be base names, not paths.`
The ignore-patterns parameter represents `Files or directories matching the regular expression patterns are skipped.The regex matches against base names, not paths.`
The ignore-paths parameter represents `Files or directories matching the regular expressions patterns to the ignore-list.`
But I use the pylint 2.11.1 version test and it doesn't work.

Reply:
I think it may be a good idea to rename `azure/cli/command_modules/network/aaz` to `aaz_generated`. It will may linter's config much easier.

Reply:
I very much agree with your idea, we should rename all the directories generated by CodeGen to aaz_generated, not only main modules, but also extensions,
But I'm not too sure how much this affects CodeGen.
We can discuss this with @kairu-ms and I'll keep trying to see if there is a way to exclude directories.